recipe/meta.yaml

```
@@ -83,8 +83,9 @@ outputs:
         - fsspec >=2021.10.1
         - aiohttp-retry >=2.4.5
         - diskcache >=5.2.1
-        - scmrepo ==0.0.16
-        - dvc-render ==0.0.4
+        - scmrepo ==0.0.19
+        - dvc-render ==0.0.5
+        - dvclive[image] >=0.7.3
```

```suggestion
        - dvclive >=0.7.3
```
the `[image]` is a requirements for dvc tests, and also not available in dvclive, if I understand correctly.

---

cc @daavoo

---

correct